tableau-databricks/oauth-config.xml

```diff
@@ -19,6 +20,10 @@
     <scopes>2ff814a6-3304-4ab8-85cb-cd0e6f879c1d/user_impersonation</scopes>
 
     <capabilities>
+        <entry>
+            <key>OAUTH_CAP_REQUIRES_PROMPT_SELECT_ACCOUNT</key>
```

Comment:
This is for always prompt user to select an account instead of directly log them in if they're already logged in.